sdk/src/main/java/com/horizen/account/api/rpc/types/TransactionArgs.java

```diff
@@ -157,8 +157,8 @@ public Message toMessage(BigInteger baseFee, BigInteger rpcGasCap) throws RpcExc
             }
         }
         return new Message(
-            from == null ? Optional.empty() : Optional.of(new AddressProposition(from.toBytes())),
-            to == null ? Optional.empty() : Optional.of(new AddressProposition(to.toBytes())),
+            getFrom(),
```

Comment:
In prev implementation we returned `Optional.empty()` in case `from` is null.
Currently, we use `getFrom()` that return `Address.ZERO` instead of `null`
Note: related to comment below.